src/sys/unix/tcp.rs

```diff
@@ -58,6 +59,50 @@ pub(crate) fn set_reuseaddr(socket: TcpSocket, reuseaddr: bool) -> io::Result<()
     )).map(|_| ())
 }
 
+pub(crate) fn get_reuseaddr(socket: TcpSocket) -> io::Result<bool> {
+    let mut optval: libc::c_int = unsafe { mem::zeroed() };
+    let mut optlen= mem::size_of::<libc::c_int>() as libc::socklen_t;
```

Comment:
Nit: ss this rustfmted? `optlen=` is incorrect.

Reply:
I've run cargo fmt and it doesn't catch it. Fixed it manually.